### PR TITLE
[Fundamentals] Updates from Social Impact portal

### PR DESCRIPTION
**Pull request by kodster28**
Copying over some of the goodness that lives at https://www.cloudflare.com/impact-portal/
+
updating Proxied DNS records to hopefully be more new-user friendly

### CODE REVIEW

**Comment by kodster28** [comment]
Open question for review... is the [new version](https://fundamentals-updates.cloudflare-docs-7ou.pages.dev/dns/manage-dns-records/reference/proxied-dns-records/) clearer than the [previous version](https://developers.cloudflare.com/dns/manage-dns-records/reference/proxied-dns-records/)?

I'd think so... but just wanting to confirm.

**Comment by ranbel** [comment]
Yes, I like how you
- moved the "When to use unproxied records" content under "Limitations"
- renamed "Unproxied records" to match what shows up in the dash

File: content/dns/manage-dns-records/reference/proxied-dns-records.md [hunk 1]
```diff
@@ -6,50 +6,52 @@ weight: 1
...
+![Proxy status affects how Cloudflare treats traffic intended for specific DNS records](/dns/static/proxy-status-screenshot.png)
 
-Because requests to proxied hostnames go through Cloudflare before reaching your origin server, these requests will appear to be coming from Cloudflare's IP addresses. You may need to adjust your server configuration to [allow Cloudflare IPs](/fundamentals/get-started/setup/allow-cloudflare-ip-addresses/).
+---
 
-{{</Aside>}}
+## Proxied records
 
-## When to proxy your DNS records
+When an `A`, `AAAA`, or `CNAME` record is **Proxied** — also known as being _orange-clouded_ — DNS queries for these will resolve to Cloudflare Anycast IPs instead of their original DNS target. This means that all requests intended for proxied hostnames will go to Cloudflare first and then be forwarded to your origin server.
```
Comment (ranbel) [inline content/dns/manage-dns-records/reference/proxied-dns-records.md]: Should orange-clouded be italicized when it's not a drop-down menu?

File: content/dns/manage-dns-records/reference/proxied-dns-records.md [hunk 2]
```diff
@@ -6,50 +6,52 @@ weight: 1
...
+Because requests to proxied hostnames go through Cloudflare before reaching your origin server, all requests will appear to be coming from Cloudflare's IP addresses (and could potentially be blocked or rate limited). If you use proxied records, you may need to adjust your server configuration to [allow Cloudflare IPs](/fundamentals/get-started/setup/allow-cloudflare-ip-addresses/).
 
 ### Limitations
 
-#### Pending domains
+#### Record types
 
-Every zone onboarded onto Cloudflare will initially be in [pending state](/dns/zone-setups/reference/domain-status/) until we can verify ownership. This means that DNS records [are not proxied](/dns/manage-dns-records/reference/proxied-dns-records/) until your zone has been activated and any requests to your DNS records will return your origin server's IP address.
+By default, Cloudflare only supports proxied `A`, `AAAA`, and `CNAME` records, you cannot proxy other record types.
```
Comment (ranbel) [inline content/dns/manage-dns-records/reference/proxied-dns-records.md]:
```suggestion
By default, Cloudflare only supports proxied `A`, `AAAA`, and `CNAME` records. You cannot proxy other record types.
```

File: content/dns/manage-dns-records/reference/proxied-dns-records.md [hunk 3]
```diff
@@ -6,50 +6,52 @@ weight: 1
...
+If you need to connect to your origin using a non-HTTP protocol (SSH, FTP, SMTP) or the traffic targets an [unsupported port](/fundamentals/get-started/reference/network-ports/) at the origin, either leave your records [unproxied (DNS-only)](#dns-only-records) or use [Cloudflare Spectrum](/spectrum/).
 
-## When to use unproxied records
+#### Pending domains
 
-In some circumstances, you should not proxy your DNS records.
+Every zone onboarded onto Cloudflare will initially be in [pending state](/dns/zone-setups/reference/domain-status/) until we can verify ownership. This means that DNS records are [DNS-only](#dns-only-records) until your zone has been activated and any requests to your DNS records will return your origin server's IP address.
 
-### A, AAAA, and CNAME records
+For enhanced security, we recommend rolling your origin IP addresses at your hosting provider after your zone has been activated. This action prevents your origin IPs from being leaked during onboarding.
```
Comment (ranbel) [inline content/dns/manage-dns-records/reference/proxied-dns-records.md:43]: what does it mean to "roll" an IP address?

**Review by ranbel — APPROVED** [review]

File: content/dns/manage-dns-records/reference/proxied-dns-records.md (hunk 3 above)
Comment (kodster28) [reply content/dns/manage-dns-records/reference/proxied-dns-records.md:43]: It's terminology we use elsewhere in the dashboard (namely, [API tokens](https://developers.cloudflare.com/api/tokens/create/#roll-api-token)).

It's equivalent to `re-generate`.

File: content/dns/manage-dns-records/reference/proxied-dns-records.md (hunk 1 above)
Comment (kodster28) [reply content/dns/manage-dns-records/reference/proxied-dns-records.md]: you're right!

**Pull request merged**